js/views/homeVw.js

```diff
@@ -449,44 +484,32 @@ module.exports = Backbone.View.extend({
     }
   },
 
-  removeSubView: function(view) {
-    var self = this,
-        index;
+  _removeSubView: function(list, view) {
```

Comment:
It's a convention to indicate that this function is private, i.e. should not be called from outside of this view. The reason is that it's expecting one of the sub view lists (itemViews or userViews) and I wouldn't want views outside of this view to have to know anything about those data structures.

I do offer more the more public friendly methods removeItemView and removeUserView, which call the "private" one, although it's probably not super likely that another view would call them.